std/range/package.d

```diff
@@ -4173,10 +4195,10 @@ private string lockstepMixin(Ranges...)(bool withIndex)
             }
             return res;
         }
-    }, params.join(", "), withIndex ? "size_t index = 0;" : "",
+    }, name, params.join(", "), indexDef,
        emptyChecks.join(" && "), dgArgs.join(", "),
        popFronts.join("\n                "),
-       withIndex ? "index++;" : "");
```

Comment:
> You're commenting on a line that's removed

Ah, I fetched last night and was seeing the old version. Looks good now.

> Why is _stoppingPolicy a runtime value anyway

I guess they figured it is more flexible and pretty cheap to check at runtime? But in this case it pushes a potential compilation failure to a runtime assertion :(